### PR TITLE
[Core] Adding bool GP GidIO postprocessing

### PR DESCRIPTION
**Pull request by loumalouomega**

### CODE REVIEW

**Review by RiccardoRossi — COMMENTED** [review]
you just added functions correct?

**Comment by loumalouomega** [comment]
Yes

File: applications/StructuralMechanicsApplication/custom_processes/spr_error_process.cpp [hunk 1]
```diff
@@ -18,6 +18,7 @@
 #include "processes/find_nodal_neighbours_process.h"
 #include "custom_processes/spr_error_process.h"
 #include "utilities/variable_utils.h"
+#include "utilities/stl_io.h"
```
Comment (jcotela) [inline applications/StructuralMechanicsApplication/custom_processes/spr_error_process.cpp]: Does this file belong in this PR?

Comment (loumalouomega) [reply applications/StructuralMechanicsApplication/custom_processes/spr_error_process.cpp]: No probably mixed

Comment (loumalouomega) [reply applications/StructuralMechanicsApplication/custom_processes/spr_error_process.cpp]: Solved

**Comment by loumalouomega** [comment]
Anyone?

**Comment by loumalouomega** [comment]
Anyone can take a look?

**Comment by loumalouomega** [comment]
Ping

**Comment by pooyan-dadvand** [comment]
Forwarding to @KratosMultiphysics/interface-committee

**Comment by loumalouomega** [comment]
@KratosMultiphysics/interface-committee This is just an extension of the current gid_io

**Comment by loumalouomega** [comment]
Ping

**Comment by jginternational** [comment]
I can't help. I know barely nothing about this

**Review by jrubiogonzalez — APPROVED** [review]

**Comment by loumalouomega** [comment]
Thanks

**Pull request merged**